src/logger/Logger.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+
+class Logger:
+    def __init__(self, debug=False, logfile='log.txt'):
+        self.debug = debug
+        self.logfile = logfile
+        self.f = open(logfile,'a+')
+
+    def __open(self):
```

Comment:
I'm not thinking it will be subclassed, I only used double underscores as I've seen that as a way of indicating that a method is intended to be private but it looks like a single underscore is the convention for that without subclassing implications.

Reply:
Yeah I think single under is enough, I don't exactly know the full implication of double under, it seems spooky that Python mangles names with double under.

Reply:
Agreed, the double does some stuff where the class name gets prepended to the method name or something weird like that.